### PR TITLE
refactor(login): redesign login page

### PR DESCRIPTION
**Pull request by ankarhem**
#### Description
add accordion component
redesign login page to use accordions with animation
one of the tasks on #646

#### Screenshot (if UI related)
Looks and functions like this, with the exception that `Sign in` is used everywhere instead of login on the local user submit button
https://streamable.com/sunpui

#### Todos

- [x] Sucessfully builds `yarn build`
- [x] Translation Keys `yarn i18n:extract`

### CODE REVIEW

**Comment by TheCatLady** [comment]
"Login" is a noun whereas "log in" is an action/verb.  Maybe you can correct that here as well?

**Comment by ankarhem** [comment]
> "Login" is a noun whereas "log in" is an action/verb. Maybe you can correct that here as well?

Had no idea! Thanks :+1: 

I'll make the change

File: src/components/Common/Accordion/index.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  AccordionContent: any;
+}
+
+export const AccordionContent: React.FC<{ isOpen: boolean }> = ({
+  isOpen,
+  children,
+}) => {
+  return (
+    <AnimateHeight height={isOpen ? 'auto' : 0}>
+      {children || <></>}
```
Comment (sct) [inline src/components/Common/Accordion/index.tsx:25]: Can we not just have this as the following?

```suggestion
      {children}
```

File: src/components/Login/LocalLogin.tsx [hunk 2]
```diff
@@ -11,17 +11,15 @@ const messages = defineMessages({
   validationemailrequired: 'Not a valid email address',
   validationpasswordrequired: 'Password required',
   loginerror: 'Something went wrong when trying to sign in',
-  loggingin: 'Logging in...',
-  login: 'Login',
-  goback: 'Go back',
+  signingin: 'Signing in...',
```
Comment (sct) [inline src/components/Login/LocalLogin.tsx:14]: Use Unicode ellipsis:
```suggestion
  signingin: 'Signing in…',
```

File: src/components/PlexLoginButton/index.tsx [hunk 3]
```diff
@@ -3,9 +3,9 @@ import PlexOAuth from '../../utils/plex';
 import { defineMessages, useIntl } from 'react-intl';
 
 const messages = defineMessages({
-  loginwithplex: 'Login with Plex',
+  signinwithplex: 'Sign in',
   loading: 'Loading...',
-  loggingin: 'Logging in...',
+  signingin: 'Signing in...',
```
Comment (sct) [inline src/components/PlexLoginButton/index.tsx:8]:
```suggestion
  signingin: 'Signing in…',
```

**Review by sct — APPROVED** [review]

**Pull request merged**